lansa-vmss-windows-autoscale-sql-database/mainTemplate.json

```diff
@@ -454,6 +487,50 @@
             }
         }
       },
+	  {
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2018-05-01",
+      "name": "dynamicSecret",
+      "properties": {
+        "mode": "Incremental",
+        "expressionEvaluationOptions": {
+          "scope": "inner"
+        },
+        "template": {
+          "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {
+			"vaultCertificate": {
+              "type": "string"
+            },
+            "vaultSecret": {
+              "type": "securestring"
+            }
+          },
+          "outputs": {
+            
+          }
+        },
+	"parameters": {
+          "vaultCertificate": {
+            "reference": {
+              "keyVault": {
+                "id": "[resourceId(subscription().subscriptionId, resourceGroup().name, 'Microsoft.KeyVault/vaults', parameters('vaultName'))]"
+			},
+              "certificateName": "[parameters('keyVaultCertificateID')]"
```

Comment:
I believce from the documentation that the certificate Name is required, as coded here. But you have asked for the Certificate ID in the UI, which is incorrect.

Reply:
Sure, will change that to "keyVaultCertificateName"